AI/Nullkiller/AIUtility.cpp

```diff
@@ -278,8 +280,8 @@ creInfo infoFromDC(const dwellingContent & dc)
 	ci.creID = dc.second.size() ? dc.second.back() : CreatureID(-1); //should never be accessed
 	if (ci.creID != -1)
 	{
-		ci.cre = VLC->creh->objects[ci.creID].get();
-		ci.level = ci.cre->level; //this is cretaure tier, while tryRealize expects dwelling level. Ignore.
+		ci.cre = static_cast<const CCreature*>(VLC->creatures()->getById(ci.creID));
```

Comment:
What prevents changing type of 'ci.cre' to whatever getById returns?

Reply:
Many functions in both AIs requires CCreature to go, I think passing Creature there will lead to another wall of changes. But will try.

Reply:
Up to you then

Reply:
done